common/roles/kubevirt/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This task should somehow be called something related to `development`.
After all installing libvirt on the host and cloning the source repository is the development flow.

Reply:
This is also non-blocking for the merge.